CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
+  * Removed parsing and generation of self-signed certificates. Also removed the platforms rp200 and wwrelay, which are no longer supported.
+  * Maestro traffic now routed through edge-proxy instead of directly connecting to the Pelion cloud.
+  * By default, disabled the Maestro's gateway logging feature in the configuration file for platforms `imx8mmevk`, `uz3eg-iocc` and `raspberrypi3`. Gateway logging feature is supported through Fluent Bit.
+* [logrotate] Added bbappend file to install a configuration that rotates the logs of files under `/var/log`.
+* [logrotate] Removed crontab method to call logrotate every 5mins. Instead, we configured systemd logrotate timer to 5mins.
+* [identity-tool] Established an independent recipe under recipes-connectivity and removed the dependency on wwrelay-utils.
+  * Converted the identity node program to bash, which lives in a new project called pe-utils.
+  * Installed v2.0.4 of identity-tool, which majorly removes the generation of self-signed certificate.
+* [edge-proxy] Configured services gateway logs (Maestro), gateway stats (Maestro), relay-term, Fluent Bit HTTP endpoint to route traffic through edge-proxy and not consume self-signed certificate.
+* [meta-nodejs] Removed the dependency on node v8.x. Upgraded the node packages to work with default nodejs version of Dunfell.
+* [relay-term] Upgraded relay-term to work with node v12.x. Established an independent recipe from wwrelay-utils and removed the dependency on global-node-modules.
+* [mbed-fcce] Upgraded factory-configurator-client-example to v4.7.0 and pinned its dependencies.
+* [fluentbit] Added recipe to install Fluentbit 1.3.5 on the gateway for providing an open source log processor and forwarder solution.
+  * By default, Fluent Bit is configured with following input endpoints - CPU, MEM, Systemd services - edge-core, edge-proxy, pelion-relay-term, maestro, kubelet, docker and wait-for-pelion-identity. And an output endpoint HTTP to publish logs at API endpoint - `http://gateways.local:8080/v3/device-logs` (routing through edge-proxy).
+* [journald] Enabled Forward Secure Sealing (FSS) feature of systemd journal. This helps detect gateway logs file tampering.
+  * To configure Pelion Edge gateway with a sealing key and to keep track of the verification key in production setup, use Pelion Edge Provisioner (PEP) tool [v2.3.0](https://github.com/PelionIoT/pelion-edge-provisioner/releases/tag/v2.3.0).
+  * By default, the gateway is configured with persistent journal logging. To disable that, set flag `VOLATILE_LOG_DIR = "no"` in `local.conf` and also update the `Storage` in recipes-core/systemd/systemd-conf/journald.conf. Note: If you disable persistent logging, the FSS feature won't work.
```

Comment:
Yocto and LmP?

Reply:
I have a new section at the top of the Changelog that specifies the only Pelion Edge feature difference.  That should address the question as to which of these bullets apply to which OS.